Bundle error: Error: Cannot find module './platform/openbsd'

I'm trying to package my program for use on another Linux computer running node.  I do not have the `openbsd` folder and I get this error:

```bash
$ echo 'require("copy-paste").copy(String(Math.random()), () =>{})' > test.js
$ browserify --bare test.js -o out.js
Error: Cannot find module './platform/openbsd' from '/home/jcalfee/eosjs/trez/node_modules/copy-paste'
    at /home/jcalfee/eosjs/trez/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/home/jcalfee/eosjs/trez/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/jcalfee/eosjs/trez/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /home/jcalfee/eosjs/trez/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:152:21)
```

I did find a work around:
```bash
browserify --bare test.js -o out.js --ignore-missing
```

What is the best way to package your module for node?

I'm having the same issue when compiling with webpack. Shouldn't this work out of the box?

```
ERROR in ./node_modules/copy-paste/index.js
Module not found: Error: Can't resolve './platform/openbsd' in '/Users/queeniema/github/myExtension/node_modules/copy-paste'
 @ ./node_modules/copy-paste/index.js 34:11-40
```

--- a/index.js
+++ b/index.js
@@ -31,7 +31,7 @@ switch(process.platform) {
 		config = require("./platform/linux");
 		break;
 	case "openbsd":
-		config = require("./platform/openbsd");
+		config = require("./platform/linux");
 		break;
 	default:
 		throw new Error("Unknown platform: '" + process.platform + "'.  Send this error to xavi.rmz@gmail.com.");

